src/whisk.rs

```diff
@@ -293,7 +295,9 @@ pub fn rand_scalar<T: RngCore>(rng: &mut T) -> Fr {
 /// Serialize field element to bytes
 pub fn to_bytes_fr(fr: &Fr) -> FieldElementBytes {
     let mut bytes = [0u8; FIELD_ELEMENT_SIZE];
-    fr.write(&mut bytes[..]).unwrap();
+
+    fr.serialize_uncompressed(&mut bytes[..]).unwrap();
```

Comment:
I left the unwrap here -- I think a general removal of unwrap for Result/Option would be better

Reply:
oversight, should use Result

Reply:
Will fix in subsequent PR addressing whisk serde